data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -161,10 +162,16 @@ public <T> void consumeRecords() throws Exception {
             topicMetrics.getNumberOfPollAuthErrors().increment();
             Thread.sleep(10000);
         } catch (RecordDeserializationException e) {
-            LOG.warn("Deserialization error - topic {} partition {} offset {}, seeking past the error record",
-                     e.topicPartition().topic(), e.topicPartition().partition(), e.offset(), e);
+            LOG.warn("Deserialization error - topic {} partition {} offset {}",
+                     e.topicPartition().topic(), e.topicPartition().partition(), e.offset());
+            if (e.getCause() instanceof AWSSchemaRegistryException) {
```

Comment:
Is this the recommended behavior for this exception? Is it meant to be retryable? Why 30 seconds?

Reply:
Because it is an exception that would require user to take some actions (change permissions to Glue Registry, etc)